dlgr/griduniverse/static/scripts/demo.js

```diff
@@ -1010,7 +1011,21 @@ function renderTransition(transition) {
     (state) => settings.item_config[state]
   );
 
-  return `✋${aStartItem.name} + ${tStartItem.name} → ✋${aEndItem.name} + ${tEndItem.name}`;
+  const aStartItemString = `✋${aStartItem ? aStartItem.name : '⬜'}`;
+  const tStartItemString = tStartItem ? tStartItem.name : '⬜';
+  if (transition_visibility == "never") {
+    return `${aStartItemString} + ${tStartItemString}`
+  }
+  
+  if (transition_visibility == "seen" && !transitionsSeen.has(transition.id)) {
+    var aEndItemString = "✋❓";
+    var tEndItemString = "✋❓";
```

Comment:
```suggestion
    var tEndItemString = "❓";
```
No hand for the target Item, I think(?)